ui/app/AppLayouts/Wallet/panels/SelectParamsForBuyCryptoPanel.qml

```diff
@@ -94,58 +86,36 @@ ColumnLayout {
             lineHeightMode: Text.FixedHeight
             verticalAlignment: Text.AlignVCenter
         }
-        TokenSelector {
-            id: holdingSelector
-            objectName: "tokenSelector"
+
+        AssetSelectorCompact {
+            id: assetSelector
+
+            objectName: "assetSelector"
+
             Layout.fillWidth: true
+
             model: root.assetsModel
-            popup.width: parent.width
-            contentItem:  Loader {
-                height: 40 // by design
-                sourceComponent: !!holdingSelector.currentTokensKey ? selectedTokenCmp : nothingSelectedCmp
-            }
-            background: StatusComboboxBackground {
-                border.width: 1
-                color: Theme.palette.transparent
-            }
-            onTokenSelected: root.tokenSelected(tokensKey)
-        }
-    }
+            sectionProperty: "sectionName"
```

Comment:
Yeah, it could be hidden, resulting with cleaner api and easier configuration of the component. It was motivated by different types of sections present in the design or no sections at all in others. The advantage is that it gives ability to easily disable sections but setting "" as section property. So it would be easy to have sectioned/not sectioned variants using the same model. But I'm not sure if it's real life scenario.